src/main/java/org/opensearch/timeseries/indices/IndexManagement.java

```diff
@@ -1037,14 +1037,8 @@ public void initFlattenedResultIndex(String flattenedResultIndexAlias, ActionLis
             choosePrimaryShards(request, false);
 
             adminClient.indices().create(request, ActionListener.wrap(response -> {
-                if (response.isAcknowledged()) {
-                    logger.info("Successfully created flattened result index: {} with alias: {}", indexName, flattenedResultIndexAlias);
-                    actionListener.onResponse(response);
-                } else {
-                    String errorMsg = "Index creation not acknowledged for index: " + indexName;
-                    logger.error(errorMsg);
-                    actionListener.onFailure(new IllegalStateException(errorMsg));
-                }
+                logger.info("Successfully created flattened result index: {} with alias: {}", indexName, flattenedResultIndexAlias);
```

Comment:
The old code seems better. Why do you remove the isAcknowledged() check?

Reply:
it's a chain of action listener calls, I don't need to handle specific isAcknowledged for inner action listener calls. It will bubble up if anything go wrong. Handling the inner action listener calls is unnecessary and cannot be covered by IT tests. The rest of methods in this file/codebase that's using a chain of action listener calls is using the same style

Reply:
added it back